nautobot/docs/models/extras/gitrepository.md

```diff
@@ -40,15 +40,18 @@ Whenever a Git repository record is created, updated, or deleted, Nautobot autom
 
 ## Repository Structure
 
++/- 2.0.0
+    Git Repositories are now imported as standard Python packages and therefore all Git Repository top level directories must now contain an `__init__.py` file.
```

Comment:
Probably good to reword this and/or merge it with the below 2.0.0 note to make it clearer that this is only a behavior/requirement if your Git repository provides Jobs.